packages/catalog-model/package.json

```diff
@@ -21,17 +21,19 @@
     "backstage"
   ],
   "scripts": {
-    "build": "backstage-cli build",
+    "build": "yarn schema:build && backstage-cli build",
```

Comment:
Tbh I'm a fan of a separate `generate` script instead, and to have the compiled schema checked into the repo. This build command isn't something that's run during development anyway, so get pretty awkward to use with `start-backend`

Reply:
I was honestly not sure how to best achieve this :D ideally it would always magically work even in dev when you change any schema file. But yes in the current state of things, it does put the generated schema into `src/generated` and it's checked in.

Reply:
Until then - yeah I can just remove the change on this line and then rely on people remembering to run the schema build when necessary and check in that too

Reply:
Would have to add a webpack loader and rollup plugin to make this magic. We could totally do that at a later point and trial behind an experimental flag, if we feel it's something that makes sense to have as part of the core platform.